### PR TITLE
[Snyk] Add Snyk pacakge

### LINKED ISSUE #654: Migrate Snyk package
# Integration release checklist

This checklist is intended for integrations maintainers to ensure consistency
when creating or updating a Package, Module or Dataset for an Integration.

### Migration Checklist

- [ ] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md)
- [ ] Supported versions of the monitoring target are documented
- [ ] Generated output for at least 1 log file exists
- [ ] Documentation explains how to configure monitoring target to work with the integration
- [ ] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [ ] At least a manual test with ES / Kibana / Agent has been performed.
- [ ] Required Kibana version set to:
- [ ] Screenshot of the "Add Integration" page on Fleet added to PR

### Dashboards

- [ ] Dashboards exists
- [ ] Screenshots added to manifest
- [ ] Data stream filters added to visualizations

### Data Streams
- audit
  - [ ] Convert edge processing to Ingest Node
  - [ ] [Pipeline tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/pipeline_testing.md) added
  - Inputs
    - httpjson
      - [ ] [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) added

    - log
      - [ ] [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) added

- vulnerabilities
  - [ ] Convert edge processing to Ingest Node
  - [ ] [Pipeline tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/pipeline_testing.md) added
  - Inputs
    - httpjson
      - [ ] [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) added

    - log
      - [ ] [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) added

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Migrates Snyk Filebeat module to Agent integration

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist

- [ ]

## How to test this PR locally
```
cd integrations/packages/snyk
elastic-package build && elastic-package stack down && elastic-package stack up --version 7.16.0-SNAPSHOT -d -v && eval "$(elastic-package stack shellinit)" && elastic-package test -v
```

## Related issues

- Closes #654

### CODE REVIEW

File: packages/snyk/data_stream/audit/_dev/test/pipeline/test-synk-audit.log [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{"groupId":"groupid123test-543123-54312sadf-123ad","orgId":"orgid123test-5643asd234-asdfasdf","userId":"userid123test-234sdfa2-423sdfa-2134","projectId":null,"event":"user.logged_in","content":{"sessionPublicId":"sessionId123-t34123-sdfa234-asd"},"created":"2020-11-17T14:30:13.800Z"}
```
Comment (efd6) [inline packages/snyk/data_stream/audit/_dev/test/pipeline/test-synk-audit.log:1]: filename: s/test-synk-audit.log/test-snyk-audit.log/

File: packages/snyk/data_stream/audit/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        required: true
+        default: 10s
+      - name: first_interval
+        type: text
+        title: Initial interval to query Snyk Events API
+        description: Go Duration syntax (eg. 10s)
+        multi: false
+        show_user: false
+        required: true
+        default: 30d
```
Comment (legoguy1000) [inline packages/snyk/data_stream/audit/manifest.yml:73]: need to change to 720h

**Review by P1llus — APPROVED** [review]

**Pull request merged**